tests/lib/at_cmd_parser/at_utils/src/main.c

```diff
@@ -53,7 +53,7 @@ static void test_command_detection(void)
 static void test_valid_notification_char_detection(void)
 {
 	for (char c = 0; c < 127; ++c) {
-		if ((c >= 'A') && (c <= 'Z')) {
+		if ((c >= 'A') && (c <= 'Z')) || (c == '_')) {
 			continue;
 		}
 
```

Comment:
@charlieshao5189 Could you add one test below for `_` being accepted as a valid notification char, e.g.
`zassert_true(is_valid_notification_char('_'), "Underscore should be a valid notification char");`

Reply:
I just added this line and lost "(", please review again.